cmd/invitation/send_invitation.go

```diff
@@ -0,0 +1,45 @@
+package invitation
+
+import (
+	"github.com/calyptia/api/types"
+	"github.com/calyptia/cli/config"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdSendInvitation(config *config.Config) *cobra.Command {
+	redirectURI := "https://core.calyptia.com"
+	if config.BaseURL == "https://cloud-api-dev.calyptia.com" {
+		redirectURI = "https://core-dev.calyptia.com"
+	}
+	if config.BaseURL == "https://cloud-api-staging.calyptia.com" {
+		redirectURI = "https://core-staging.calyptia.com"
+	}
+
+	cmd := &cobra.Command{
+		Use:   "invitation EMAIL", // child of `calyptia create`
+		Short: "Send an invitation to a user to join the current project",
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			for _, email := range args {
+				err := config.Cloud.CreateInvitation(ctx, config.ProjectID, types.CreateInvitation{
+					Email:       email,
+					RedirectURI: redirectURI,
+				})
+				if err != nil {
+					cmd.PrintErrf("failed to send invitation for %q: %v\n", email, err)
+					continue
+				}
+
+				cmd.Printf("invitation sent to %q successfully\n", email)
+			}
+
+			return nil
+		},
+	}
+
+	fs := cmd.Flags()
+	fs.StringVar(&redirectURI, "redirect-uri", redirectURI, "Redirect URI for the invitation, leave the default value if you don't know what it is")
```

Comment:
Where are you using this value? You should set it to "https://core.calyptia.com" and use redirectURI